codes/classical/bits/binary_linear.yml

```diff
@@ -3,12 +3,15 @@
 ##       https://github.com/errorcorrectionzoo       ##
 #######################################################
 
-code_id: binary_linear
-physical: bits
-logical: bits
 
-name: 'Binary linear code'
-#introduced: ''
+
+code_id: 'binary_linear'
+
+physical: 'bits'
+
+logical: 'bits'
+
+name: 'Linear binary code'
 
 description: 'An \((n,2^k,d)\) linear code is denoted as \([n,k,d]\). Its codewords form a linear subspace, i.e., for any codewords \(x,y\), \(x+y\) is also a codeword. Often defined in terms of a \textit{generator matrix} \(G\), whose rows form a basis for the \(k\)-dimensional codespace. Given a codeword \(x\), the corresponding encoded codeword is \(G^T x\).'
```

Comment:
The other part of this assignment was adding the definition of Tanner graph. Please do so next round, below the defns of G and H.